misc/README.md

```diff
@@ -2,15 +2,6 @@
-| Locally confirm Indexer E2E tests pass with new inputs.<br />* _Warning_:  Only use these commands locally because they're _not_ secure.<br />* The commands directly invoke `e2elive.py` to increase local iteration speed. `make e2e` spins up a Docker compose environment. <br><b>* If you are running into dependency issues please consult the e2elive.py prerequisites section below</b></br>| * `docker run -it --rm --name some-postgres -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_USER=$USER -e POSTGRES_DB=indexer_db_e2e postgres` <br/>* `python misc/e2elive.py --source-net $E2EDATA/net_done.tar.bz2 --connection-string "host=localhost port=5432 user=$USER dbname=indexer_db_e2e sslmode=disable"`<br /> * Look for `e2elive.py` output starting with: `indexer e2etest OK`.                                                                                                                                                            |
-| Upload test run artifacts to S3.<br/>* Publishing to S3 streamlines running the tests for other developers and CI.<br />* If you don't have an AWS account, request artifact upload by pinging Will, Michael, or DevOps.                                                       | Run the s3 copy command provided by `misc/buildtestdata.sh` output.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-| `make e2e` runs on every PR. Validate results on the next CI run.                                                                                                                                                                                                              | * Find the most recent workflow starting _after_ updating the S3 bucket:  [https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all](https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all).<br/>* If successful, confirm the test used the artifact you uploaded.  Click a job link, expand _make e2e_ and search for `INFO:util:s3://`.<br />* If unsuccessful, investigate source of error.  If additional help is needed, raise in #indexer.  It is _your_ responsibility to ensure the build is fixed. |
-
 ## e2elive.py prerequisites
 `e2elive.py` requires access to algorand-indexer and goal binaries. If you receive an error related to the algorand-indexer binary then you need to build the indexer from scratch by running `make` in the root of the indexer repository. This will insert a binary in the `cmd/algorand-indexer` directory, where `e2elive.py` expects it by default.
```

Comment:
@AlgoStephenAkiki _If_ https://github.com/algorand/indexer/pull/956#discussion_r847871513 is resolved without further changes to the PR, then I think the highlighted section can also be removed.  It's only referenced by the deleted instructions.

```suggestion

```